diff -Naur a/drv/brew/brew.go b/drv/brew/brew.go
--- a/drv/brew/brew.go	2026-05-18 11:55:00.000000000 +0700
+++ b/drv/brew/brew.go	2026-06-17 16:14:21.076407163 +0700
@@ -60,7 +60,7 @@
 		return &final_output, err
 	}
 
-	cli = []string{up.BrewPath, "upgrade"}
+	cli = []string{up.BrewPath, "upgrade", "-y"}
 	out, err = session.RunUID(up.Config.Logger, slog.LevelDebug, up.BaseUser, cli, up.Config.Environment)
 	tmpout = CommandOutput{}.New(out, err)
 	tmpout.Context = "Brew Upgrade"
